### ISSUE TITLE
Workbox swDest references are inconsistent

### ISSUE THREAD

**Issue by philipwalton** [google/WebFundamentals#7369]
Reported on twitter: https://twitter.com/marcushellberg/status/1103827852552736768

@jeffposnick it looks like many of our build tool docs share config partials. I believe `workbox-webpack-plugin` is the only one that defaults to a  particular `swDest` option. Can you confirm?

If so, any objection to just changing all of our `swDest` examples to be `service-worker.js`?

**Comment by jeffposnick** [comment]
That's correct; only the webpack plugin has a default `swDest` filename—because webpack can infer the output path. The other build tools need you to provide output info.
